X16 Reference - Appendix F - 65C816 Processor.md

```diff
@@ -0,0 +1,2517 @@
+
+---
+
+### DEY
+
+**Decrement .Y**
+
+```text
+SYNTAX           MODE       HEX LEN CYCLES      FLAGS   
+DEY              imp        88  1   2           n.....z. .
+```
+
+Decrement .X The **z** flag is set when the value reaches zero. This
+makes DEC, DEX, and DEY useful as a loop counter, by setting the number of
+iterations, the repeated operation, then DEX followed by BNE.
+
+**z** is set when the counter reaches zero.
+**n** is set when the high bit gets set.
+
+
+[[Opcodes](#instructions-by-opcode)] [[By Name](#instructions-by-name)] [[By Category](#instructions-by-category)]
+
+---
+
+### EOR
+
+**Exclusive OR**
+
+```text
+SYNTAX           MODE       HEX LEN CYCLES      FLAGS   
+EOR #$20/#$1234  imm        49  3-m 3-m         n.....z. .
+EOR $20          dir        45  2   4-m+w       n.....z. .
+EOR $20,X        dir,X      55  2   5-m+w       n.....z. .
+EOR $20,S        stk,S      43  2   5-m         n.....z. .
+EOR $1234        abs        4D  3   5-m         n.....z. .
+EOR $1234,X      abs,X      5D  3   6-m-x+x*p   n.....z. .
+EOR $1234,Y      abs,Y      59  3   6-m-x+x*p   n.....z. .
+EOR $FEDBCA      long       4F  4   6-m         n.....z. .
+EOR $FEDCBA,X    long,X     5F  4   6-m         n.....z. .
+EOR ($20)        (dir)      52  2   6-m+w       n.....z. .
+EOR ($20),Y      (dir),Y    51  2   7-m+w-x+x*p n.....z. .
+EOR ($20,X)      (dir,X)    41  2   7-m+w       n.....z. .
+EOR ($20,S),Y    (stk,S),Y  53  2   8-m         n.....z. .
+EOR [$20]        [dir]      47  2   7-m+w       n.....z. .
+EOR [$20],Y      [dir],Y    57  2   7-m+w       n.....z. .
+```
+
+Perform an Exclusive OR operation with the operand and .A
+
+EOR compares each bit of the operands and sets the result bit to 1 if one of the
+two bits is 1. If both bits are 1, the result is 0. If both bits are 0, the
+result is 0.
+
+EOR is useful for _inverting_ the bits in a byte. `EOR #$FF` will flip an entire
+byte. (This will always flip the low byte in .A. To flip both bytes when **m**
+is 0, you would use `EOR #$FFFF`.)
+
+Truth table for EOR:
+
+```text
+Operand 1: 1100
+Operand 2: 1010
+Result:    0110
+```
+
+
+[[Opcodes](#instructions-by-opcode)] [[By Name](#instructions-by-name)] [[By Category](#instructions-by-category)]
+
+---
+
+### INC
+
+**Increment**
+
+```text
+SYNTAX           MODE       HEX LEN CYCLES      FLAGS   
+INC              acc        1A  1   2           n.....z. .
+INC $20          dir        E6  2   7-2*m+w     n.....z. .
+INC $20,X        dir,X      F6  2   8-2*m+w     n.....z. .
+INC $1234        abs        EE  3   8-2*m       n.....z. .
+INC $1234,X      abs,X      FE  3   9-2*m       n.....z. .
+```
+
+Increment .A or memory
+
+Adds 1 to the value in .A or the specified memory address. The **n** and **z**
+flags are set, based on the resultant value.
+
+INC is useful for reading strings and operating on large areas of memory,
+especially ith indirect and indexed addressing modes.
```

Comment:
typo

Reply:
I looked at that 4 times before I saw the "ith".